jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/language/languages/CodeWhispererJavaScript.kt

```diff
@@ -4,10 +4,13 @@
 package software.aws.toolkits.jetbrains.services.codewhisperer.language.languages
 
 import software.aws.toolkits.jetbrains.services.codewhisperer.language.CodeWhispererProgrammingLanguage
+import software.aws.toolkits.jetbrains.services.codewhisperer.util.FileCrawler
+import software.aws.toolkits.jetbrains.services.codewhisperer.util.JavascriptCodeWhispererFileCrawler
 import software.aws.toolkits.telemetry.CodewhispererLanguage
 
 class CodeWhispererJavaScript private constructor() : CodeWhispererProgrammingLanguage() {
     override val languageId: String = ID
+    override val fileCrawler: FileCrawler = JavascriptCodeWhispererFileCrawler
```

Comment:
all the JS-related extensions and file crawlers should live in jetbrains-ultimate